src/main/kotlin/blackjack/domain/participant/state/Cards.kt

```diff
@@ -0,0 +1,21 @@
+package blackjack.domain.participant.state
+
+import blackjack.domain.deck.Card
+import blackjack.domain.deck.CardNumber
+
+class Cards(
+    val values: List<Card> = listOf(),
+) {
```

Comment:
외부에 List 인터페이스의 메소드들이 노출되긴 하겠지만 불변이라 아래와 같이 delegation 사용을 고려해볼 수 있을 것 같네요.

```suggestion
class Cards(values: List<Card> = listOf()) : List<Card> by values {
```

노출되는 타입도 일관성 있게 Cards로 가져갈 수 있을 것 같아요.

```kotlin
fun cards(): Cards
```

Reply:
'`List<Card>`를 delegation 한다는게 어떤 의미지? 캡슐화가 깨지지 않을까?' 고민 했는데,
제 코드를 다시 뜯어보고서야 규남님의 깊은 뜻을 이해할 수 있었습니다.. 🙇‍♂️ 
어차피 `values: List<Card>`를 getter 처럼 끌어다 사용해서 캡슐화고 뭐고 없으니,
delegation 해서 `List<Card>` 대신 `Cards`로 들고 다니면 어떠냐는 말씀이셨던거 같아요!
굉장히 멋진 코드인거 같아서 당장 반영했습니다 👍 

<img width="1276" alt="image" src="https://user-images.githubusercontent.com/37354145/177244911-e510b577-d867-4ca5-a13d-ceeb008d1f10.png">

다만 `List<Card>`를 최종적으로 하는 곳들이 모두 출력을 위해 List 컬렉션을 내보내고 있어서
최후에는 values를 꺼내어 사용하는게 아쉽네요... 😭